src/app/PCM/index.js

```diff
@@ -14,12 +14,15 @@ import { maxRecursiveDepth } from 'utils/constants'
  * `PRI_TEMPLATE_CODE`, using the pcm stored at `code` in the redux store.
  */
 
-const Pcm = ({ code, properties, depth }) => {
+const Pcm = ({ code, properties, depth, config }) => {
   const depthPlusOne = addOne(depth)
   const mappedPcm = useGetMappedPcm(code)
   const { PRI_TEMPLATE_CODE: templateCode } = mappedPcm
   const parentCode = code
-  const template = templateHandlerMachine(depthPlusOne)(mappedPcm)(parentCode)(properties)
+  const template = templateHandlerMachine(depthPlusOne)(mappedPcm)(parentCode)({
```

Comment:
Instead of passing config and properties as an object in the function invocation itself, let's create an object called configuration which has config and properties as it's key and pass it as an argument to the function invocation of templateHandlerMachine.

ie: let configuration = {properties, config}